### DISCUSSION TITLE
Folders in Library

### DISCUSSION THREAD

**Discussion by Yell038** [keiwando/toonsquid#328] [Ideas]
animation clips/symbols is a great feature, love it! but we DEFINITELY need something to organize it in the library, like folders! Seriously, finding your clips through the other tons you have is a real pain.

the search bar is fine but it’d be nice to have folders imo

thats all!

**Comment by Cat-Butler** [comment]
Interim workaround (I suspect you know this already, @Yell038 - but I thought I'd mention it for any new forum readers): from version 1.9, we can now save all those symbols externally, arranged in folders and sub-folders, via Files. (Also good for moving between iPads, team workflow, backup, importing only the symbols needed into a project.)
Major drawback of this workaround: unlike in the ToonSquid internal Library, this does not allow visual browsing 😢- it relies on meaningful naming.

**Comment by pravdomil** [comment]
see https://github.com/keiwando/toonsquid/discussions/256
